Src/Plugins/Security/SEC1001.SecurePlaintextSecrets.json

```diff
@@ -40,7 +40,7 @@
         },
         {
           "SubId": "SlackToken",
-          "ContentsRegex": "xox[pboa]-[0-9]{12}-[0-9]{12}-[0-9]{12}-[0-9a-z]{32}",
+          "ContentsRegex": "(?<fingerprint>xox[pboa]-[0-9]{12}-[0-9]{12}-[0-9]{12}-[0-9a-z]{32})",
```

Comment:
>fingerprint [](start = 31, length = 11)

same here, don't need this. i will now stop remarking on this condition, would be good to scrub for it.